apps/ledger-live-desktop/src/renderer/families/solana/TransactionConfirmFields.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useMemo } from "react";
+import { getDeviceTransactionConfig } from "@ledgerhq/live-common/transaction/index";
+import { SolanaFamily } from "./types";
+import Alert from "~/renderer/components/Alert";
+import { Trans } from "react-i18next";
+import ConfirmTitle from "~/renderer/components/TransactionConfirm/ConfirmTitle";
+import Box from "~/renderer/components/Box";
+import { openURL } from "~/renderer/linking";
+import { useLocalizedUrl } from "~/renderer/hooks/useLocalizedUrls";
+import { urls } from "~/config/urls";
+import { DeviceModelId } from "@ledgerhq/devices";
+import { Link } from "@ledgerhq/react-ui";
+
+const Title: TitleComponent = props => {
+  const { transaction, account, parentAccount, status, device } = props;
+  const transferTokenHelpUrl = useLocalizedUrl(urls.solana.splTokenInfo);
+
+  const fields = getDeviceTransactionConfig({
+    account,
+    parentAccount,
+    transaction,
+    status,
+  });
+
+  const typeTransaction: string | undefined = useMemo(() => {
+    const typeField = fields.find(field => field.label && field.label === "Type");
+
+    if (typeField && typeField.type === "text" && typeField.value) {
+      return typeField.value;
+    }
+  }, [fields]);
+
+  if (
+    transaction.model.commandDescriptor?.command.kind === "token.transfer" &&
+    device.modelId === DeviceModelId.nanoS
+  ) {
+    return (
+      <Box
+        flexDirection="column"
+        alignItems="center"
+        gap={4}
+        mb={4}
+        justifyContent="center"
+        className="title-test-transfer"
```

Comment:
Do we need this ?

Reply:
Not sure, I think I reused the component or it's maybe from the initial dev, let me check

Reply:
It doesn't seem to be used, I removed it